Neos.Media.Browser/Classes/Controller/AssetController.php

```diff
@@ -379,22 +316,89 @@ public function replaceAssetResourceAction(Asset $asset)
     }
 
     /**
-     * Edit an asset
+     * Show an asset
      *
-     * @param Asset $asset
+     * @param string $assetSourceIdentifier
+     * @param string $assetProxyIdentifier
      * @return void
+     * @throws StopActionException
+     * @throws UnsupportedRequestTypeException
      */
-    public function editAction(Asset $asset)
+    public function showAction(string $assetSourceIdentifier, string $assetProxyIdentifier)
     {
-        $this->view->assignMultiple([
-            'tags' => $asset->getAssetCollections()->count() > 0 ? $this->tagRepository->findByAssetCollections($asset->getAssetCollections()->toArray()) : $this->tagRepository->findAll(),
-            'asset' => $asset,
+        if (!isset($this->assetSources[$assetSourceIdentifier])) {
+            throw new \RuntimeException('Given asset source is not configured.', 1509702178632);
+        }
+
+        $assetBrowser = $this->assetSources[$assetSourceIdentifier]->getAssetProxyRepository();
```

Comment:
In all other places this variable is called `$assetProxyRepository`

Reply:
good catch, that's a leftover from the old code.